src/base-app.js

```diff
@@ -68,6 +68,12 @@ class FusionApp {
         },
       };
     }
+    token.stacks.push({type: 'register', stack: new Error().stack});
```

Comment:
I'm getting TypeErrors that stacks is undefined. Can we add a `token.stacks || token.stacks = []` guard?